### ISSUE TITLE
Download links for OpenSuse are not working

### ISSUE THREAD

**Issue by frankbroniewski** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#1224]
## Download Links for OpenSuse Leap and SLE lead to a 404
Hello,

the links on the download page for OpenSuse Leap and SLE do not work anymore, they result in a 404.

Leap links are nowadays:
- https://download.opensuse.org/repositories/Application:/Geo/15.5/
- https://download.opensuse.org/repositories/Application:/Geo/15.6/

Be aware though that there are no QGIS packages for Leap in the repositories!  And I could not find any replacement links for SLE. 

Page URL: https://qgis.org/en/site/forusers/alldownloads.html#suse-opensuse

**Comment by agiudiceandrea** [comment]
Thanks. There is an open PR https://github.com/qgis/QGIS-Website/pull/1196 by @trytomakeyouprivate about the SUSE / OpenSUSE install instructions. The latest updates of the links have been made by @badarotti.

**Comment by frankbroniewski** [comment]
Hey Andrea, thank you for the information. The OpenSuse installation has another issue - there is no QGIS python module available. I created an issue on the QGIS issue tracker for that, since I think this belongs technically more there than here: https://github.com/qgis/QGIS/issues/56401

If this is intentional - plugins do work - this should maybe reflected somewhere, as this makes plugin development a tad more tricky ...

**Comment by agiudiceandrea** [comment]
@frankbroniewski if the QGIS plugins work, then the `qgis` python module is available.

**Comment by frankbroniewski** [comment]
Hey Andrea, thanks for your help - the non standard Python module path on OpenSuse caught me off guard, since it is different to any other distro I used (Fedora and Ubuntu) before. I closed the corresponding ticket on QGIS with some more comments, if you are interested.

On a side note, I looked through the changes of the install [documentation](https://github.com/qgis/QGIS-Website/pull/1196/commits/e2edd74ae3f119cd0cabd99f09175c74be22247d#diff-b1cadc89d24da7a18e53486f114d4097fdf7ef22f9ae7d045b2c24f2d6bc9f02R120) you linked above and noted the `pip install` command. This is deprecated on OpenSuse Tumbleweed and other distros, as they implement [PEP668](https://peps.python.org/pep-0668/) in the current version. The system/user wide installation of Python modules outside of the system repositories is discouraged. Instead users should install Python modules in a virtual environment, i.e. by using `pipx` [Link](https://github.com/pypa/pipx) instead of `pip`.

Here's an example output of the error you get when using pip:
```
frank@schnipsel:~> pip install flask
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try
    zypper install python311-xyz, where xyz is the package
    you are trying to install.
    
    If you wish to install a non-rpm packaged Python package,
    create a virtual environment using python3.11 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip.
    
    If you wish to install a non-rpm packaged Python application,
    it may be easiest to use `pipx install xyz`, which will manage a
    virtual environment for you. Install pipx via `zypper install python311-pipx` .

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

I know that Fedora and Debian also follow PEP668 already, for others I do not know.

If I may suggest, I would rewrite the [paragraph](https://github.com/qgis/QGIS-Website/pull/1196/commits/e2edd74ae3f119cd0cabd99f09175c74be22247d#diff-b1cadc89d24da7a18e53486f114d4097fdf7ef22f9ae7d045b2c24f2d6bc9f02R118) as follows:

> If Plugins report missing packages, you can install them using one of the following methods in preferred order:  
>1. Look in your system repositories and install them via the system package manager, i.e. apt, dnf or alternative: `sudo apt install module-name`  
>2. If your system implements PEP668, use pipx to install the module: `pipx install module-name`  
>3. Lastly, use pip to install the missing module: `pip install module-name`

If you want, I can do a PR with my proposal, I just need to figure out what I need to do 😁

**Comment by agiudiceandrea** [comment]
@frankbroniewski, thanks for sharing such info. Your help is appreciated, so please add a comment to the PR https://github.com/qgis/QGIS-Website/pull/1196 and / or open a new PR.

**Comment by frankbroniewski** [comment]
Hey Andrea, please see https://github.com/qgis/QGIS-Website/pull/1196#issuecomment-1953828980 for a link to the PR. I let you decide whether to close this ticket or leave it open 👍

**Comment by agiudiceandrea** [comment]
@frankbroniewski the download links for OpenSUSE are actually mostly incorrect, so this issue report is valid.

**Comment by boredsquirrel** [comment]
I am not sure why we should add links to individual packages tbh. This is unnecessary (the Debian instructions are insanely huge) and people should really really use their package manager for that.

**Comment by frankbroniewski** [comment]
Yes, I agree, especially with the Debian/Ubuntu instructions, and it doesn't even mention `add-apt-repository` for a more convenient way to add a foreign repository.

I even think for most users, the Flatpak route should be the default, if their system repository does not have a recent enough version of QGIS. Just for developing plugins, Flatpak is not ideal in regards to code completion and stuff - don't know where the Flatpak version stores its plugins - could be difficult for easy replacement during development.

Imo, a more user-friendly way to describe the QGIS installation on Linux would be to 
1. Point to the system's graphical software manager tool. This would limit Ubuntu users to the QGIS version in their repository, as they don't have Flatpak installed by default. All other software managers on other distros should already propose the Flathub version afaik.
2. Propose Flatpak as the second viable option if the version in the package manager does not suffice (too old). Once Flatpak is enabled, QGIS should turn up in their graphical software manager.
3. Provide alternative, distro-specific resources, but keep the installation tips to a minimum.

For Ubuntu for example, it could suffice to provide a link to the PPA, which in turn has already instructions on how to add the PPA to one's system.

But maybe this discussion belongs in its own issue and not here ...
Concerning who is providing QGIS packages for distros, this is rather opaque to me. Does the QGIS project build packages for distros and makes them available in the system repositories? Or how does that work?

**Comment by boredsquirrel** [comment]
afaik python was broken in the flatpak, it would be best to have that fixed though. But yeah different thing.
